namui-animation-editor/src/types/history_system/history.rs

```diff
@@ -4,16 +4,6 @@ pub struct History<TState> {
     states: Vec<TState>,
 }
 
-#[derive(Debug, PartialEq)]
-pub enum UndoError {
-    NothingToUndo,
-}
-
-#[derive(Debug, PartialEq)]
-pub enum RedoError {
-    NothingToRedo,
-}
-
```

Comment:
Changes about returning result on undo is reverted. Is it intended?

Reply:
What the hell

Reply:
Fix it